Langref polish (#2193)
slint-ui/slint: docs/langref/src/repetition.md

# Repetition

> The `for`-`in` syntax will add an element multiple times.

Reviewer: ```suggestion
Use the `for`-`in` syntax to create an element multiple times.
```
